### DISCUSSION TITLE
Migration support in sync fork bunnet

### DISCUSSION THREAD

**Discussion by PetrF0X** [BeanieODM/beanie#408] [General]
Hi @roman-right,
Since there's no Discussions section on the new [bunnet](https://github.com/roman-right/bunnet) project I'm going to ask here - is there a plan for supporting migrations with the sync API?

I'm asking because we are introducing support for data migration and it is only supported by the async beanie (we use the sync package). I'm thinking what's the best course of action here - wait for support in bunnet (how long would it take?) or using both projects for now.

Thank you! Petr

**Comment by roman-right** [comment]
Hey!
I plan to add migrations to the Bunnet project by the end of the week. The PR is already there: https://github.com/roman-right/bunnet/pull/4

P.S.: I opened discussions there. Thank you for the information :)
